homeassistant/components/switcher_kis/__init__.py

```diff
@@ -157,6 +158,11 @@ def device_id(self) -> str:
         """Switcher device id."""
         return self.data.device_id  # type: ignore[no-any-return]
 
+    @property
+    def device_key(self) -> str:
+        """Switcher device key."""
+        return self.data.device_key  # type: ignore[no-any-return]
+
```

Comment:
This is not used anywhere in the code, no need to add this property
```suggestion
```

Reply:
You use the `data` --> `self.coordinator.data.device_key`, this would be accesbile using `self.coordinator.device_key` which is not needed in this case.
Coordinator properties are used if the underling data needs to be processed and accessed in multiple places.

Reply:
Try the remove the code as I suggested above and noticed that it is not used anywhere

Reply:
it is done.